### PR TITLE
Use -Wl so linkopts only use a single argument

### PR DESCRIPTION
**Pull request by segiddins**
This ensures they wont be incorrectly de-duplicated if put into a NestedSet

### CODE REVIEW

File: apple/internal/linking_support.bzl [hunk 1]
```diff
@@ -89,7 +85,7 @@ def _register_linking_action(ctx, extra_linkopts = []):
     rpaths = rule_descriptor.rpaths
     linkopts = []
     if rpaths:
-        linkopts.extend(collections.before_each("-rpath", rpaths))
+        linkopts.extend(["-Wl,-rpath,{}".format(rpath) for rpath in rpaths])
```
Comment (keith) [inline apple/internal/linking_support.bzl:88]:
```suggestion
        linkopts.append("-Wl,-rpath,{}".format(rpath) for rpath in rpaths)
```

**Review by keith — COMMENTED** [review]
this seems legit but I wonder if there's ever a difference by doing this vs letting clang forward them

File: apple/internal/linking_support.bzl (hunk 1 above)
Comment (segiddins) [reply apple/internal/linking_support.bzl:88]: how would this work? don't we need extend since we're adding multiple elements?

Comment (keith) [reply apple/internal/linking_support.bzl:88]: oh sorry you're right, you still need extend, but at least in py you wouldn't need to form the list normally

Comment (keith) [reply apple/internal/linking_support.bzl:88]: turns out starlark doesn't support this mode

Comment (keith) [inline apple/internal/linking_support.bzl:88]: you can remove this `if` now

**Review by keith — APPROVED** [review]

**Comment by keith** [comment]
https://github.com/bazelbuild/rules_apple/pull/973

**Pull request closed without merging**